command/certificate/create.go

```diff
@@ -24,6 +24,7 @@ const (
 	profileSelfSigned     = "self-signed"
 	profileIntermediateCA = "intermediate-ca"
 	profileRootCA         = "root-ca"
+	profileDirect         = "direct" // Used only on sign
```

Comment:
I think we should use `csr` here. I think that matches better with the implementation in `certificates` which uses csrTemplate, etc.